rites/roman1969/src/calendars/countries/hong-kong/index.ts

```diff
@@ -0,0 +1,81 @@
+import { PatronTitle } from '../../../constants/martyrology-metadata';
+import { Precedences } from '../../../constants/precedences';
+import { CalendarDef } from '../../../models/calendar-def';
+import { Inputs } from '../../../types/calendar-def';
+
+export class HongKong extends CalendarDef {
+  particularConfig = {
+    ascensionOnSunday: true,
+    epiphanyOnSunday: true,
+    corpusChristiOnSunday: true,
+  };
+
+  inputs: Inputs = {
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
```

Comment:
I can’t this celebration listed on that website at all, nor can I see a list of holy days of obligation on that website. Please provide a real source for this.

[Here](http://catholic-dlc.org.hk/st/bydate.htm#:~:text=God-,%2A%E9%81%87%E4%B8%8A,%E7%B5%90%E6%9D%9F), however, I can see `遇上主日，仍然舉行`, but I am not sure how to interprete it:

- is it only celebrated if it occurs/falls on Sunday? (I think that is unlikely)
- is it always transferred to Sunday? if yes, what if it occurs with Epiphany? what are the exact transfer rules?

Reply:
> * is it only celebrated if it occurs/falls on Sunday? (I think that is unlikely)

yes, it is what it meant. that why it is not holy days of obligation in Hong Kong unless it's on Sunday. and It will not transfer.

Reply:
> that why it is not holy days of obligation in Hong Kong unless it's on Sunday. and It will not transfer.

As I understand this statement of yours is that `mary_mother_of_god` is always inscribed as solemnity on 1 Jan, but it is never a holy day of obligation (unless it occurs on Sunday which is always a holy day of obligation), therefore, the celebration is never dropped (which would `only celebrated` suggest): it is always celebrated, but it is never a holy day of obligation.

If this interpretation is correct, then the implementation is correct and the only change we should make then is to update the source comment.

```suggestion
    // src: http://catholic-dlc.org.hk/st/bydate.htm#:~:text=God-,%2A%E9%81%87%E4%B8%8A,%E7%B5%90%E6%9D%9F
```